iwyu_tool.py

```diff
@@ -197,41 +225,31 @@ def slice_compilation_db(compilation_db, selection):
 
     return new_db
 
+def execute(invocations, verbose, formatter, jobs):
+    if jobs == 1:
```

Comment:
Please cut this block, it's not necessary as mentioned earlier.

Reply:
I don't want busy-waiting when there's only a single job in flight though.